ui/app/styles/core/tag.scss

```diff
@@ -52,4 +52,14 @@
   &.is-outlined {
     box-shadow: 0 0 0 1px $white;
   }
+
+  &.is-structure {
+    color: rgb(98, 104, 115);
```

Comment:
The colour specified in Structure isn’t present in Bulma. This is equivalent to `hsl(219, 8%, 42%)`. `$grey-dark` is `hsl(0, 0%, 29%)` and `$grey-light` is `hsl(0, 0%, 71%)`, neither seemed close enough. Is there a plan for how to handle transitioning to Structure? Maybe it makes more sense to use an existing colour for now and revisit it when we are migrating components.

Reply:
Probably a question for @DingoEatingFuzz this but (my 2 cents), why not just drop a structure colors file into you CSS folder somewhere obvious, then slowly migrate over. Once structure becomes a package, the ui color names (`red/blue/gray/green/etc-000`) will remain the same, so you'll just be able to remove the color file from wherever you put it and use the node_modules/package one instead. Me personally, I'd rather start migrating over and use var names than use `rgb(98, 104, 115)`. Remember any colors you don't use will just get compiled out, and the new structure naming scheme shouldn't overwrite anything in Bulma, or anything older in the nomad codebase.

Here's ours https://github.com/hashicorp/consul/blob/master/ui-v2/app/styles/base/color/base-variables.scss

Looks like vault is doing similar: https://github.com/hashicorp/vault/blob/master/ui/app/styles/utils/_colors.scss

I think TF is doing something similar.

I think this is generally the approach folks outlined in RFC-land also, again obvs check with @DingoEatingFuzz.